skimage/measure/fit.py

```diff
@@ -573,6 +573,10 @@ def ransac(data, model_class, min_samples, residual_threshold,
         where the probability (confidence) is typically set to a high value
         such as 0.99, and e is the current fraction of inliers w.r.t. the
         total number of samples.
+    exceptions : exception class or tuple of exception classes
```

Comment:
Perhaps turn exceptions into warnings for now, but mention in the API changes that we're tightening this to throw by default.